specification/paths/Organizations.json

```diff
@@ -52,7 +54,9 @@
                           "attributes": {
                             "type": "object",
                             "required": [
-                              "name"
+                              "name",
+                              "billing_address",
```

Comment:
I do not know if this creates issues with onboarding.

Reply:
It may work with our current onboarding, but that is a temporary solution.

This change may limit our onboarding options in the future.